src/snowflake/snowpark/modin/plugin/_internal/aggregation_utils.py

```diff
@@ -1034,9 +1034,9 @@ def generate_column_agg_info(
             The new index data column index names for the dataframe after aggregation
     """
 
-    quoted_identifier_to_snowflake_type: dict[
+    quoted_identifier_to_type: dict[
```

Comment:
as we discussed offline, let's remove this renaming out of this PR. We can revisit this naming decision in future PRs.